scripts/just.config.ts

```diff
@@ -40,7 +40,19 @@ export const build = () => {
               if (args.importer) {
                 let extPath = args.path;
                 if (extPath.startsWith(".")) {
-                  extPath = extPath + ".mjs";
+                  const absolutePath = path.resolve(
+                    args.importer,
+                    "..",
+                    extPath
+                  );
+                  if (
+                    fs.existsSync(absolutePath) &&
+                    fs.lstatSync(absolutePath).isDirectory()
+                  ) {
+                    extPath = extPath + path.sep + "index.mjs";
```

Comment:
After some thinking, we may prefer to use `/` here vs `path.sep`. Otherwise there could be potential problems with character escaping depending on how this will be used later. Windows should handle the `/` just fine.

Reply:
Yup, right so you are 👍